src/lib/profile-normalised-data.js

```diff
@@ -0,0 +1,87 @@
+import { database_pool } from './database.js';
+import Settings from './settings.js';
+import apply_data_profile from './util-data-profile.js';
+
+
+
+async function profile_normalised_data_all() {
+    for(var profile_name of Settings.dataProfiles) {
+        // not await - run each profile at once
+        profile_normalised_data_all_for_profile(profile_name);
+    };
+}
+
+async function profile_normalised_data_all_for_profile(profile_name) {
+
+    const select_sql = 'SELECT normalised_data.* FROM normalised_data '+
+        'LEFT JOIN normalised_data_profile_results '+
+        'ON normalised_data_profile_results.normalised_data_id = normalised_data.id AND normalised_data_profile_results.profile_name=$1 '+
+        'WHERE normalised_data_profile_results.normalised_data_id IS NULL AND normalised_data.data_deleted=FALSE '+
+        'ORDER BY normalised_data.updated_at ASC LIMIT 10';
+
+    while(true) {
+
```

Comment:
I usually try to make sure there is some kind of safety catch in a while true, might be worth adding if there is way to do that

Reply:
Point - The catch() block should have break in it - that will apply in several places tho so I'll do that in another P.R.